test/functional/api/v2/compliance/arf_reports_controller_test.rb

```diff
@@ -436,6 +436,30 @@ class Api::V2::Compliance::ArfReportsControllerTest < ActionController::TestCase
     assert_equal report.id, response['results'].first["id"].to_i
   end
 
+  test "should order by compliance_[failed|passed|othered]" do
+    reports_cleanup
+    policy = FactoryBot.create(:policy)
+    create_arf_report_for_search({ "passed" => 4, "othered" => 0, "failed" => 1 }, policy)
+    create_arf_report_for_search({ "passed" => 1, "othered" => 0, "failed" => 0 }, policy)
+    create_arf_report_for_search({ "passed" => 15, "othered" => 9, "failed" => 0 }, policy)
+    create_arf_report_for_search({ "passed" => 2, "othered" => 3, "failed" => 7 }, policy)
+
+    get :index, :params => { :order => "compliance_failed DESC" }, :session => set_session_user
+    assert_response :success
+    response = ActiveSupport::JSON.decode(@response.body)
+    assert_equal 7, response['results'].first['failed']
+
+    get :index, :params => { :order => "compliance_passed DESC" }, :session => set_session_user
+    assert_response :success
+    response = ActiveSupport::JSON.decode(@response.body)
+    assert_equal 2, response['results'].first['passed']
```

Comment:
We'd expect 15, but due to default scope https://github.com/theforeman/foreman_openscap/blob/master/app/models/foreman_openscap/arf_report.rb#L30, it will always order by latest time creation first.